src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+// @ts-ignore Module is written in Flow
+import { useFeatureConfig } from '../common/feature-checking';
+
+// @ts-ignore Module is written in Flow
+import DropdownMenu from '../../components/dropdown-menu';
+import SidebarNavSignButton from './SidebarNavSignButton';
+import SignMe32 from '../../icon/fill/SignMe32';
+import SignMeOthers32 from '../../icon/fill/SignMeOthers32';
+import { Menu, MenuItem } from '../../components/menu';
+
+// @ts-ignore Module is written in Flow
+import messages from './messages';
+// @ts-ignore Module is written in Flow
+import { SIDEBAR_NAV_TARGETS } from '../common/interactionTargets';
+
+import './SidebarNavSign.scss';
+
+export function SidebarNavSign() {
+    const {
+        blockedReason: boxSignBlockedReason,
+        onClick: onBoxClickRequestSignature,
+        onClickSignMyself: onBoxClickSignMyself,
+        status: boxSignStatus,
+        targetingApi: boxSignTargetingApi,
+        isSignRemoveInterstitialEnabled,
+    } = useFeatureConfig('boxSign');
+
+    return (
+        <>
+            {isSignRemoveInterstitialEnabled ? (
+                <DropdownMenu isResponsive constrainToWindow isRightAligned>
+                    <SidebarNavSignButton
+                        blockedReason={boxSignBlockedReason}
+                        status={boxSignStatus}
```

Comment:
we only have design for the tooltip "Request Signature". I don't have enough context why we are doing this because this logic was already there, but I'm noticing EUA is not sending any status, so it will be always `undefined`

Reply:
I think it's better to create a separate ticket and dig a little bit more about this